### PR TITLE
Make File Now Accepts Multiple Flags

### LINKED ISSUE #112: Add the ability to parse multiple arguments in make file
Currently we can parse the `dropdata` flag, but we should be able to iterate over all arguments.  The first new one would be debug.

`--debug`:
sets `-debug=true`

### PR DESCRIPTION
**Pull request by Hopding**
Fixes #112.

### CODE REVIEW

**Comment by thebho** [comment]
Just to be clear, this allows for a `--dropdata`, `--nodebug` or both flags and works as excepted?

File: make [hunk 1]
```diff
@@ -1,56 +1,74 @@
...
+echo "Running skilldirectoryschema..."
 docker exec -it cassandra_container bash usr/bin/cqlsh -f /data/skilldirectoryschema.cql
 echo "Schema update complete"
 
+### Run SkillDirectory executable - note that the executable isn't run in the docker
+### container. The container only holds the Cassandra database that the executable
+### connects to.
 echo "Running Skill Directory..."
-./skilldirectory
+if $debug_flag; then # Run with debug-level logging, unless "--nodebug" flag was used
```
Comment (thebho) [inline make]: Its better practice to have only one execute command.  This should still work if you do `./skilldirectory -debug=$debug_flag`

**Review by thebho — CHANGES_REQUESTED** [review]

File: make (hunk 1 above)
Comment (Hopding) [reply make]: Changes made - it does work that way, and looks much nicer too 👍

**Comment by Hopding** [comment]
Correct. You can specify either flag or both, and it will work as expected either way.

**Review by thebho — APPROVED** [review]

**Pull request merged**